lib/Runtime/Language/DynamicProfileInfo.h

```diff
@@ -84,6 +84,31 @@ namespace Js
         }
     };
 
+    struct CallbackInfo
+    {
+        bool CanInlineCallback(Js::ArgSlot argIndex)
+        {
+            return canInlineCallback && argNumber == argIndex;
+        }
+
+        // False if there is more than one ArgIn that is a function object or a function object with arg number greater than MaxInlineeArgoutCount
+        Field(uint8) canInlineCallback : 1;
+
+        Field(uint8) isConstructor : 1;
```

Comment:
unused?

Reply:
I added that before looking into what/how that tracks. thanks for catching that.